shardingsphere-proxy/shardingsphere-proxy-bootstrap/pom.xml

```diff
@@ -93,6 +93,11 @@
             <artifactId>shardingsphere-cluster-mode-repository-etcd</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.shardingsphere</groupId>
+            <artifactId>shardingsphere-cluster-mode-repository-nacos</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
I need to remove it because there is no license declare for Nacos and its dependencies